pkg/apis/management.cattle.io/v3/globaldns_types.go

```diff
@@ -64,6 +64,7 @@ type Route53ProviderConfig struct {
 	RoleArn         string `json:"roleArn,omitempty"`
 	Region          string `json:"region" norman:"default=us-east-1"`
 	ZoneType        string `json:"zoneType" norman:"default=public"`
+	BatchChangeSize string `json:"batchChangeSize" norman:"default=100"`
```

Comment:
I think instead of adding this specific field only to the config, it would be useful to add a Values map[string]string to catch all such needs of custom external-dns deployment. 
Since we anyway need to change UI to expose this - we might as well expose a key-value text field to user and append them to answers during the chart deployment.

Reply:
Such request to add some fields has been made before and it requires us to change Rancher API and UI - accepting key-value pairs would solve such usecases.

Reply:
Also better to add this key-value input for all providers along with route53

Reply:
updated